src/main/kotlin/io/bazel/worker/CpuTimeBasedGcScheduler.kt

```diff
@@ -0,0 +1,41 @@
+package io.bazel.worker
+
+import com.sun.management.OperatingSystemMXBean
+import java.lang.management.ManagementFactory
+import java.time.Duration
+import java.util.concurrent.atomic.AtomicReference
+
+class CpuTimeBasedGcScheduler(
```

Comment:
Mirroring what's being done inside the generic Bazel worker

https://github.com/Bencodes/bazel/blob/3835d9b21ad524d06873dfbf465ffd2dfb635ba8/src/main/java/com/google/devtools/build/lib/worker/WorkRequestHandler.java#L431-L474

Reply:
It maybe worthwhile for future readers to inline the link to bazel source and part of the PR description in a class comment :)